src/extensionkit.cs

```diff
@@ -44,8 +44,11 @@ interface EXHostViewControllerDelegate {
 		[Export ("hostViewControllerWillDeactivate:error:")]
 		void WillDeactivate (EXHostViewController viewController, [NullAllowed] NSError error);
 
+#if !XAMCORE_5_0
+		[Obsoleted (PlatformName.MacOSX, 14, 0, message: "Not longer required as of macOS 14.0")]
```

Comment:
```suggestion
		[Obsoleted (PlatformName.MacOSX, 14, 0, message: "No longer required as of macOS 14.0")]
```
or maybe 'Not required ..."